src/components/SignIn.js

```diff
@@ -72,13 +57,9 @@ export default function SignIn() {
   // end of snackbar
 
   // user authentication
-  const [loginError, setLoginError] = React.useState(false);
-  let errorMessage = '';
 
   // error handler function
   const handleLoginError = (error) => {
```

Comment:
we still need something to open the snackbar when there's an erorr. there's a few ways we can do this. Option 1 is to open the snackbar manually by changing the `open` state here in the error handler where we know an error has occured:

```
const handleLoginError = (error) => {
  setOpen(true);
}
```